--- mappers/mappers.go
+++ mappers/mappers.go
@@ -34,17 +34,12 @@

-// Acronymify creates an acronym by taking the first letter of each part, converting it to an UpperCaseWord
-func Acronymify(parts []strings2.Part) []strings2.Part {
+// Acronymify creates an acronym by taking the first letter of each part, converting it to an UpperCaseWord
+func Acronymify(parts []strings2.Part) []strings2.Part {
	var b strings.Builder
	for _, p := range parts {
		if p == nil {
			continue
		}
		if _, ok := p.(*strings2.SeparatorPart); ok {
			continue
		}
-		s := p.String()
-		if len(s) > 0 {
-			r, _ := utf8.DecodeRuneInString(s)
-			b.WriteRune(unicode.ToUpper(r))
-		}
+		subs := p.SubParts()
+		for _, sp := range subs {
+			if sp.IsLetter() {
+				b.WriteRune(unicode.ToUpper(sp.Rune()))
+				break
+			}
+		}
	}
	if b.Len() > 0 {
