--- types.go
+++ types.go
@@ -328,10 +328,6 @@
		switch o := opt.(type) {
		case Option:
			o(cfg)
-		case WordMapper:
-			if o != nil {
-				words = o(words)
-			}
		}
	}

@@ -572,7 +568,7 @@

	for _, o := range opts {
		switch v := o.(type) {
-		case Option, WordMapper:
+		case Option:
			fmtOpts = append(fmtOpts, v)
		case ParserOption, Partitioner, PartitionerConfig:
			parseOpts = append(parseOpts, v)
