--- cli/main.go
+++ cli/main.go
@@ -9,7 +9,7 @@
	"github.com/arran4/strings2"
 )

-func process(input string, output string, args []string, opts []any, fn func(string, ...any) (string, error)) {
+func process(input string, output string, args []string, fn func(string, ...any) (string, error), opts ...any) {
	var in io.Reader
	if input == "-" {
		in = os.Stdin
@@ -83,7 +83,7 @@
 //	args: ... String to convert if file/stdin not provided
 func Camel(input string, output string, delimiter string, screaming bool, whispering bool, firstUpper bool, firstLower bool, mixCaseSupport bool, noSmartAcronyms bool, numberSplitting bool, strict bool, args ...string) {
	opts := buildOpts(delimiter, screaming, whispering, firstUpper, firstLower, mixCaseSupport, noSmartAcronyms, numberSplitting, strict)
-	process(input, output, args, opts, strings2.ToCamel)
+	process(input, output, args, strings2.ToCamel, opts...)
 }

 // Snake is a subcommand `strings2 snake`
@@ -103,7 +103,7 @@
 //	args: ... String to convert if file/stdin not provided
 func Snake(input string, output string, delimiter string, screaming bool, whispering bool, firstUpper bool, firstLower bool, mixCaseSupport bool, noSmartAcronyms bool, numberSplitting bool, strict bool, args ...string) {
	opts := buildOpts(delimiter, screaming, whispering, firstUpper, firstLower, mixCaseSupport, noSmartAcronyms, numberSplitting, strict)
-	process(input, output, args, opts, strings2.ToSnake)
+	process(input, output, args, strings2.ToSnake, opts...)
 }

 // Kebab is a subcommand `strings2 kebab`
@@ -123,7 +123,7 @@
 //	args: ... String to convert if file/stdin not provided
 func Kebab(input string, output string, delimiter string, screaming bool, whispering bool, firstUpper bool, firstLower bool, mixCaseSupport bool, noSmartAcronyms bool, numberSplitting bool, strict bool, args ...string) {
	opts := buildOpts(delimiter, screaming, whispering, firstUpper, firstLower, mixCaseSupport, noSmartAcronyms, numberSplitting, strict)
-	process(input, output, args, opts, strings2.ToKebab)
+	process(input, output, args, strings2.ToKebab, opts...)
 }

 // Pascal is a subcommand `strings2 pascal`
@@ -143,6 +143,6 @@
 //	args: ... String to convert if file/stdin not provided
 func Pascal(input string, output string, delimiter string, screaming bool, whispering bool, firstUpper bool, firstLower bool, mixCaseSupport bool, noSmartAcronyms bool, numberSplitting bool, strict bool, args ...string) {
	opts := buildOpts(delimiter, screaming, whispering, firstUpper, firstLower, mixCaseSupport, noSmartAcronyms, numberSplitting, strict)
-	process(input, output, args, opts, strings2.ToPascal)
+	process(input, output, args, strings2.ToPascal, opts...)
 }
