--- types.go	2023-10-24 10:00:00.000000000 +0000
+++ types.go	2023-10-24 10:00:00.000000000 +0000
@@ -324,8 +324,13 @@
	cfg := &caseConfig{delimiter: "-"}

	for _, opt := range opts {
-		if o, ok := opt.(Option); ok {
-			o(cfg)
+		switch o := opt.(type) {
+		case Option:
+			o(cfg)
+		case WordMapper:
+			if o != nil {
+				words = o(words)
+			}
		}
	}

@@ -566,7 +571,7 @@

	for _, o := range opts {
		switch v := o.(type) {
-		case Option:
+		case Option, WordMapper:
			fmtOpts = append(fmtOpts, v)
		case ParserOption, Partitioner, PartitionerConfig:
			parseOpts = append(parseOpts, v)
